testbook/tests/conftest.py

```diff
@@ -0,0 +1,46 @@
+from typing import List, Optional
+
+import pytest
+from jupyter_client import kernelspec
+from nbformat.notebooknode import NotebookNode
+from nbformat.v4 import new_notebook, new_code_cell, new_output
+
+
+@pytest.fixture
+def notebook_factory():
```

Comment:
Awesome! 👍 

I used to spawn a new jupyter server every time I had to create a new notebook for testing. This way we can provide the cells in the test module itself and create the notebook in memory. Thanks for this!

Reply:
You're welcome